node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const desiredFile = process.argv[2];
+
+console.log(desiredFile);
+
+fs.readFile(desiredFile, (err, data) => {
+  if (err) throw (err);
+  console.log(data.toString());
```

Comment:
(Optional) `.toString()` is nice and careful here. However, console.log doesn't require it to be a string so you could leave it off.